stack.yaml

```diff
@@ -1,5 +1,4 @@
-flags: {}
 packages:
```

Comment:
pls correct me if I'm wrong, but I believe it should be:
```
packages: []
```
otherwise I got error:
```
Error while parsing $.library.dependencies - expected Array, Object, or String, encountered Null
```

```
$ hpack --version
hpack version 0.28.2
```